src/modules/markets/helpers/get-outcome-name.js

```diff
@@ -0,0 +1,22 @@
+import {
+  SCALAR,
+  YES_NO,
+  YES_NO_YES_OUTCOME_NAME
+} from "modules/common-elements/constants";
+
+export default function getOutcomeName(market, outcomeId) {
+  const { outcomes, marketType, scalarDenomination } = market;
```

Comment:
this should be on the market object set by market.js selector. I don't think we need this helper used all over the place.